internal/container/container.go

```diff
@@ -20,11 +20,13 @@ import (
 	"log/slog"
 	"os"
 	"os/exec"
+	"path/filepath"
 	"strings"
 )
 
 func Generate(ctx context.Context, language, apiRoot, apiPath, output, generatorInput string) error {
-	return runGenerate(apiRoot, fmt.Sprintf("google-cloud-%s", language), apiPath)
+	languageDir := filepath.Join(output, fmt.Sprintf("google-cloud-%s", language))
+	return runGenerate(apiRoot, languageDir, apiPath)
```

Comment:
This logic is really just perpetuating what was already there, but I'm guessing this is not the behavior that we'll choose to keep.

Reply:
1. We probably want to pass this in as an argument at some point down the line

Reply:
Sounds good